src/components/new-image/new-image.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class NewImage extends React.Component {
+  render() {
+    const { props } = this;
+    return (
+      <div
+        style={{
+          padding: '5px',
+          width: `${props.size}px`,
+          height: `${props.size}px`
+        }}
+      >
+        <div className={`bg-${props.color || 'blue'}-light px3 round-full`}>
+          <svg viewBox="0 0 18 18" className={`color-${props.color || 'blue'}`}>
```

Comment:
Recommend adding the `mt3` class to this svg element to add 3 pixels to the top to make the icon a little more centered in the circle:

```suggestion
          <svg viewBox="0 0 18 18" className={`color-${props.color || 'blue'} mt3`}>
```

Result:

![image](https://user-images.githubusercontent.com/2180540/62467695-56a76600-b762-11e9-9374-6e72993715fc.png)

Reply:
Ooh, yes! Much prettier!